extensions/wikia/FandomCreatorEmail/Controller/ContentUpdatedController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace FandomCreatorEmail\Controller;
+
+use FandomCreatorEmail\FandomCreatorEmailController;
+
+class ContentUpdatedController extends FandomCreatorEmailController {
+	/** @var string */
+	private $contentTitle;
+
+	/** @var string */
+	private $contentUrl;
+
+	public function initEmail() {
+		parent::initEmail();
+		$this->contentTitle = $this->request->getVal( 'contentTitle' );
+		$this->contentUrl = $this->request->getVal( 'contentUrl' );
+	}
+
+	public function getSubject(): string {
+		return $this->magicWordWrapper->wrap( function() {
+			return $this->getMessage(
+					'emailext-watchedpage-article-edited-subject',
+					$this->contentTitle,
+					$this->getCurrentUserName()
```

Comment:
We're not going to call this directly. We're going to create a controller that takes some sort of secret that will create async tasks to send the emails. See the other tickets in -2040.